src/timeAgo.js

```diff
@@ -21,21 +21,17 @@ angular.module('yaru22.angular-timeago', [
       });
     }
   };
-}]).factory('nowTime', ['$window', '$rootScope', function ($window, $rootScope) {
-  var nowTime = Date.now();
-  var updateTime = function() {
-    $window.setTimeout(function() {
-      $rootScope.$apply(function(){
-        nowTime = Date.now();
-        updateTime();
-      });
-    }, 1000);
+}]).factory('nowTime', function ($timeout) {
```

Comment:
Could you make this change a separate PR? Want this PR to be strictly about Chinese language support.